config/puma.rb

```diff
@@ -31,7 +31,7 @@
 # before forking the application. This takes advantage of Copy On Write
 # process behavior so workers use less memory.
 #
-preload_app!
+# preload_app!
```

Comment:
Is it worth adding a comment to say that you can't do this due to Elastic-apm?

Reply:
It's in the commit message ;)